HW6/Lecture6.cpp

```diff
@@ -0,0 +1,141 @@
+﻿
+#include <iostream>
+#include <cctype>
+using namespace std;
+int main()
+{
+	//Task 1
+	cout << "Task I" << endl;
+	int a1;
+	int d;
+	int an;
+	cout << "Enter initial number: ";
+	cin >> a1;
+	cout << "Enter step of progression: ";
+	cin >> d;
+	cout << "Enter last number: ";
+	cin >> an;
+	for (int i = a1; i <= an; i = i + d)
+	{
+		cout << i << endl;
+	}
```

Comment:
**an** it is not the last number of progression. It is the number of the last element